buildSrc/src/main/kotlin/kotest-multiplatform-conventions.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+   id("kotlin-conventions")
+   id("kotest-jvm-conventions")
+   id("kotest-js-conventions")
+}
+
+kotlin {
+   targets {
+      linuxX64()
+
+      mingwX64()
+
+      macosX64()
+      macosArm64()
+
+      tvos()
+      tvosSimulatorArm64()
+
+      watchosArm32()
+      watchosArm64()
+      watchosX86()
+      watchosX64()
+      watchosSimulatorArm64()
+
+      iosX64()
+      iosArm64()
+      iosArm32()
+      iosSimulatorArm64()
```

Comment:
TODO: Option to configure targets, all multiplatform targets should still be default.